conf/silhouette.conf

```diff
@@ -3,26 +3,39 @@ silhouette {
   authenticator.headerName = "X-Auth-Token"
   authenticator.issuerClaim = "play-angular-silhouette"
   authenticator.encryptSubject = true
-  authenticator.authenticatorExpiry = 43200
+  authenticator.authenticatorExpiry = 12 hours
 
   # OAuth1 token secret provider settings
   oauth1TokenSecretProvider.cookieName="OAuth1TokenSecret"
   oauth1TokenSecretProvider.cookiePath="/"
   oauth1TokenSecretProvider.secureCookie=false // Disabled for testing on localhost without SSL, otherwise cookie couldn't be set
   oauth1TokenSecretProvider.httpOnlyCookie=true
-  oauth1TokenSecretProvider.expirationTime=300
+  oauth1TokenSecretProvider.expirationTime=5 minutes
+
+  # OAuth2 state provider settings
```

Comment:
OAuth2 state isn't needed server side because it's handled client side by Satellizer